server/src/main/java/org/elasticsearch/action/admin/indices/get/TransportGetIndexAction.java

```diff
@@ -68,18 +65,6 @@ public TransportGetIndexAction(TransportService transportService, ClusterService
         this.indexScopedSettings = indexScopedSettings;
     }
 
-    @Override
-    protected String executor() {
-        // very lightweight operation, no need to fork
-        return ThreadPool.Names.SAME;
```

Comment:
There is no need to overwrite this method, because the super class uses the same thread pool.